cli/cmd/project/members.go

```diff
@@ -55,7 +55,7 @@ func ListMembersCmd(cfg *config.Config) *cobra.Command {
 
 func AddCmd(cfg *config.Config) *cobra.Command {
 	addCmd := &cobra.Command{
-		Use:   "add",
+		Use:   "add <project-name> <email> <role>",
```

Comment:
Can we also show a list of valid roles ?

Reply:
It should also be provided as part of error message. Invalid Role `blabla` should be one of `admin`, `collaborator`, `viewer`